docs/index.rst

```diff
@@ -29,8 +29,15 @@ The module is designed around Pandas, Numpy and Matplotlib. This makes it easy
 to use while offering lots of possibilities for data analysis and
 visualization.
 
-FastF1 handles big chunks of data (~50-100mb per session) so most of the
-information is stored locally as cached requests (be aware).
+FastF1 handles big chunks of data (~50-100mb per session). To improve performance
+data is per default cached locally as requests (be aware). The default placement
+of the cache is operating system specific:
+
+  - Windows: `%LOCALAPPDATA%\Temp\fastf1`
+  - macOS: `~/Library/Caches/fastf1`
+  - Linux: `~/.cache/fastf1` if `~/.cache` exists otherwise `~/.fastf1`
+
+You can override the default location by using `Cache.enable_cache('location')`.
```

Comment:
I'd rather put this information into the docstring of the `Cache` class. If a user allready knows that `Cache` exists (or a search engine directs them there), they'll expect the information there. And we should try to not duplicate information in the docs for easier maintenance. So I'd rather phrase is something like "The default placement of the cache is operating system specific. A custom location can be set if desired. For more information see :class:\`~fastf1.api.Cache\`".
Then add the detailed explanation there, including the information about the environment variable.